website/content/docs/platform/k8s/injector/index.mdx

```diff
@@ -189,6 +189,27 @@ The configuration map must contain either one or both of the following files:
 
 An example of mounting a Vault Agent configmap [can be found here](/vault/docs/platform/k8s/injector/examples#configmap-example).
 
+### Injector telemetry
+
+Vault Agent Injector collects the following Prometheus metrics (along with the
+default set of golang metrics):
+
+- `vault_agent_injector_request_queue_length` - A gauge of webhook requests in
+  the injector's queue.
```

Comment:
```suggestion
- `vault_agent_injector_request_queue_length` - The number of pending webhook requests for the injector.
```

"A gauge" doesn't really describe the metric. Is it the total number of pending requests? The average number of queued requests over an interval?

Reply:
Sure, "number of pending webhook requests" is clearer.